src/backend/joanie/signature/backends/lex_persona/client.py

```diff
@@ -0,0 +1,414 @@
+"""Base Signature with Lex Persona Backend"""
+import json
+import os
+import re
+import time
+from logging import getLogger
+from typing import Dict, List
+
+import requests
+from rest_framework import status
+from rest_framework.response import Response
+
+from joanie.core.utils import webhooks
+from joanie.signature import exceptions
+
+from ..base import BaseSignatureBackend
+
+logger = getLogger(__name__)
+
+
+# pylint: disable=too-many-instance-attributes
+class LexPersonaClient(BaseSignatureBackend):
+    """
+    The Signature base class.
+    It contains generic methods to trigger
+    the workflow process of signing a document,
+    and to retrieve it once it has been signed.
+    """
+
+    name = "lex_persona"
+
+    def __init__(self, configuration):
+        """
+        Access data must be kept secret.
+        Replace those values in your .env file with your
+        own credentials provided by Lex Persona.
+        """
+        super().__init__(configuration)
+        self._set_configuration_attributes(configuration)
+        created, validity = self._set_create_and_validity_date()
+        self.workflow_id = str()
```

Comment:
?

Reply:
I have update the __init__ of the client class of the signature provider : 
```python
    def __init__(self, configuration=None, contract=None):
        """
        Access data must be kept secret.
        Replace those values in your .env file with your
        own credentials provided by Lex Persona.
        """
        super().__init__(configuration)
        self.contract = contract
        conf = configuration.get("configuration", {})
        self.base_url = conf.get("base_url", None)
        self.token = conf.get("token", None)
        self.session_user = conf.get("session_user", None)
        self.signature_page = conf.get("signature_page", None)
        self.headers = {
            "Authorization": f"Bearer {self.token}",
            "Content-Type": "application/json",
        }
        self.consent_page = conf.get("consent_page", None)

        created, validity = self._set_create_and_validity_date()
        self.workflow_id = str()
```